versions/ufo3/glyphs/glif.md

```diff
@@ -167,6 +167,18 @@ This element may occur any number of times.
 
 #### This element has no child elements.
 
+#### Anchor naming conventions
+
+The following are agreed upon standard naming conventions for the `name` of anchors for special uses.
+
+##### Ligature hcaret and vcaret
+
+Anchors with a `name` starting with `hcaret` or `vcaret` indicate horizontal or vertical position for ligature glyphs, for example to complie the [OpenType GDEF Ligature Caret List table]. Anchors with the name `hcaret` provide the `x` value for a ligature caret, and the `y` value is ignored. Likewise, anchors with the name `ycaret` provide the `y` values for a ligature caret, and the `x` value is ignored.
```

Comment:
- As per the discussion, I would love for `hcaret` to be renamed to `caret`, and that we require the anchor name to start with `caret_` and `vcaret_` respectively, in both occurances.
- "complie" -> "compile"
- "ycaret" -> "vcaret"